pkg/build/helm.go

```diff
@@ -137,6 +140,12 @@ func sanitizeChart(manifest model.Manifest, s string) error {
 				contents = strings.ReplaceAll(contents, before, after)
 			}
 
+			if s == path.Join(manifest.RepoDir("istio"), baseChart) && fname == "values.yaml" {
+				before := `defaultRevision: ""`
+				after := "defaultRevision: default"
```

Comment:
why does this happen in this repo instead of values.yaml in istio/istio?

Reply:
After a little bit more tracking this change was done when the validating webhook was moved from discovery to base. I'd prefer we go with that change. Let me experiment with that as it'd be closer to where we'd want it to live.